src/test/java/api/support/fixtures/NoticeMatchers.java

```diff
@@ -42,6 +42,10 @@ private NoticeMatchers() {
     tokenMatchers.put("item.callNumber", notNullValue(String.class));
     tokenMatchers.put("item.materialType", notNullValue(String.class));
     tokenMatchers.put("item.loanType", notNullValue(String.class));
+    tokenMatchers.put("item.effectiveLocationSpecific", notNullValue(String.class));
```

Comment:
Are there some tests which check that the values are correct?

Reply:
No there is no matching for exact values. Item resource which passed as method parameter does not contain contain the location information itself. So I found it hard to implement matching for exact values and decided just to check it for presence. Is it acceptable?

Reply:
I shall leave that decision up to you.

Personally, I think there should be (not necessarily API) tests that check that the values are what was expected. Otherwise, any value could be used and the test would pass.